Bottom: 39b2644b928e6002722322c3bc49dd41a8071926
Top:    aae556ac1ed01bfdea396c3a4ec8bc6e78667bbf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-24 22:01:59 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
